memory/includes/common/mem.hpp

```diff
@@ -8,6 +8,34 @@ constexpr uint64_t SHIFT_KB = 10ULL;
 constexpr uint64_t SHIFT_MB = 20ULL;
 constexpr uint64_t SHIFT_GB = 30ULL;
 
+template <int8_t shift, typename T = uint32_t>
```

Comment:
why this functions if we have the same in common/bitops.h

Reply:
Removed, thanks